test/Non1788tests/hyperbolic.jl

```diff
@@ -27,12 +27,12 @@ end
 @testset "cosh_rev_test" begin
     @test isapprox(cosh_rev(∅, -∞..∞)[2], ∅)
     @test isapprox(cosh_rev(Interval(-10.0, -1.0), -∞..∞)[2], ∅)
-    @test isapprox(cosh_rev(Interval(0.0, Inf), -∞..∞)[2], Interval(0.0, ∞))
+    @test isapprox(cosh_rev(Interval(0.0, Inf), -∞..∞)[2], Interval(-∞, ∞))
```

Comment:
Why is `isapprox` being used here? We should be testing equality I think?

Reply:
These tests were not added by me, I have only edited them to return negative values as well.
Should I replace `isapprox` too?